tests/components/airvisual/test_config_flow.py

```diff
@@ -295,9 +287,7 @@ async def test_step_reauth(hass, config_entry):
 
     new_api_key = "defgh67890"
 
-    with patch("homeassistant.config_entries.ConfigEntries.async_reload"), patch(
-        "pyairvisual.air_quality.AirQuality.nearest_city"
-    ):
+    with patch("homeassistant.config_entries.ConfigEntries.async_reload"):
```

Comment:
Usually we patch `async_setup_entry` in the integration instead of Home Assistant itself